scripts/checklistMove.js

```diff
@@ -0,0 +1,23 @@
+$(".get").on("click",function(){
+    console.log(this.value);
+
+    let $sideCheck = $("#"+this.value);
+    if($sideCheck.attr("checked")){
+        $sideCheck.attr("checked",false);
+    }
+    else{
+        $sideCheck.attr("checked",true)
+    }
+});
+
+$(".side").on("click",function(){
```

Comment:
Add a comment for anonymous function.

For PEAR standards brackets should be next line for function